AdSpecterPlugin/Class1.cs

```diff
@@ -435,6 +496,11 @@ public void DetectClickThrough()
                 }
             }
         }
+
+        public string GetCallToAction()
+        {
+            return adUnitWrapper.ad_unit.call_to_action;
```

Comment:
Check against null case when accessing a potentially null adUnitWrapper

If null or empty string, do not show CTA bar